torq.q

```diff
@@ -144,7 +144,11 @@ checkdependency:{[path]
         {[t;dict] runchk[dict;t;]'[t[`dependency]]}[;dict]'[t]]}
 
 getconfig:{[path;level]
-        /-check if KDBAPPCONFIG exists
+        /- check if KDBSERVCONFIG exists
+        keyappconf:$[not ""~ksc:getenv[`KDBSERVCONFIG];
```

Comment:
This will be overwritten by `keyappconf` on line 152, can remove the squares from `getenv`.